dynamodb/src/main/scala/zio/dynamodb/DynamoDBQuery.scala

```diff
@@ -551,27 +552,37 @@ object DynamoDBQuery {
     )
   }
 
+  final case class BatchWriteItemResponse(
+    unprocessedItems: Option[MapOfSet[TableName, BatchWriteItem.Write]]
+  )
+
   private[dynamodb] final case class BatchWriteItem(
     requestItems: MapOfSet[TableName, BatchWriteItem.Write] = MapOfSet.empty,
     capacity: ReturnConsumedCapacity = ReturnConsumedCapacity.None,
     itemMetrics: ReturnItemCollectionMetrics = ReturnItemCollectionMetrics.None,
-    addList: Chunk[BatchWriteItem.Write] = Chunk.empty
-  ) extends Constructor[Unit] { self =>
+    addList: Chunk[BatchWriteItem.Write] = Chunk.empty,
+    retryAttempts: Int = 0,
```

Comment:
I could not find where this is set anywhere? I guess the code has not been called yet.

Reply:
Yea right now I don't have any callers for it. I was going to add a helper to set it. Maybe something like `withRetryAttempts(Int)` -- (maybe a good use case for zio-prelude for positive only numbers?) and `withRetryTimeout(Duration)`?